packages/web-components/src/donut-chart/donut-chart.ts

```diff
@@ -0,0 +1,316 @@
+import { attr, FASTElement, nullableNumberConverter } from '@microsoft/fast-element';
+import { arc as d3Arc, pie as d3Pie } from 'd3-shape';
+import { createTabster, getMover, getTabsterAttribute, MoverDirections } from 'tabster';
+import { getDataConverter } from '../utils/chart-helpers.js';
+import { IChartProps } from './donut-chart.options.js';
+
+const tabsterCore = createTabster(window);
+getMover(tabsterCore);
+
+export class DonutChart extends FASTElement {
+  @attr({ converter: nullableNumberConverter })
+  public height: number = 200;
+
+  @attr({ converter: nullableNumberConverter })
+  public width: number = 200;
+
+  @attr({ attribute: 'hide-legends', mode: 'boolean' })
+  public hideLegends?: boolean;
+
+  @attr({ attribute: 'hide-tooltip', mode: 'boolean' })
+  public hideTooltip?: boolean;
+
+  @attr({ converter: getDataConverter('donut-chart') })
+  public data!: IChartProps;
+
+  @attr({ attribute: 'inner-radius', converter: nullableNumberConverter })
+  public innerRadius: number = 1;
+
+  @attr({ attribute: 'value-inside-donut' })
+  public valueInsideDonut?: string;
+
+  private _selectedLegend: string = '';
+
+  constructor() {
+    super();
+  }
+
+  private bindEvents() {}
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.render();
+  }
+
+  render() {
+    const rootDiv = document.createElement('div');
+    this.shadowRoot?.appendChild(rootDiv);
+    rootDiv.classList.add('root');
```

Comment:
If this is a root level div you can remove and just use `:host` as your css selector. The template can have multiple root template siblings.

Reply:
The dom element manipulation changes are being worked on.
We have run into a bug where svg content is not working as expected in fluent web components and will continue to be handled in js till we find a solution. https://github.com/microsoft/fast/issues/6599.